### ISSUE TITLE
Finicky crash when opening ZeroNet

### ISSUE THREAD

**Issue by valimero** [johnste/finicky#171]
# Finicky crash when opening ZeroNet

Hello ! 

I have no problem while using Finicky unless when I launched the app ZeroNet.

Normally when starting ZeroNet it opens a page to http://127.0.0.1:43110

## Version:
- Finicky: **v3.3.1 (282)**
- macOS: **10.15.7**
- ZeroNet: **0.6.5 (rev3852)**

### Requirements:
- Finicky installed
- ZeroNet installed : https://zeronet.io
### Actions:
- Start Finicky
- Start ZeroNet (Finicky crashed by shutting down)

**Comment by valimero** [comment]
I launched the app from the source code and i have this error in the console:
```
2021-02-09 18:44:00.989268+0100 Finicky[81168:1149763] Fatal error: Unexpectedly found nil while unwrapping an Optional value: file Finicky/Application.swift, line 20
```

**Comment by johnste** [comment]
This happens when finicky can't get the bundleIdentifier or the name from the application opening the url. I've included a fix in a1af0126626405c5bc69499d5e1da9456e02d29e which will be available in the next release

**Comment by johnste** [comment]
Newest release is out now, hopefully this issue is fixed https://github.com/johnste/finicky/releases/tag/v3.4.0
